### ISSUE TITLE
Tracker for keyboard layout problems

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#192]
```
Currently there are some problems with non-English keyboard bindings.
Depending on the platform, some of the problems listed below might not apply

1. French keyboard: mapping of keys 1-5 (issue 148 and issue 142)
2. Some Euro-keyboards: the ` key is not present (issue 120)
3. Russian/French keyboard: the confirmation key ('Y' in the English layout) is 
not mapped to the physical 'Y' key

All above are SDL-dependent bugs but we should try to find some workaround 
while a new version (SDL-1.3) is being released.
```

-----
Original issue reported on code.google.com by `vittorio...@gmail.com` on 24 Feb 2011 at 9:26
* Blocked on: #111, #120, #138, #142, #148, #152, #155, #317, #479

**Comment by GoogleCodeExporter** [comment]
```
4. Spanish keyboard: doesn't print special characters correctly (issue 152)
```

-----
Original comment by `vittorio...@gmail.com` on 24 Feb 2011 at 10:01

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `vittorio...@gmail.com` on 24 Feb 2011 at 11:41

**Comment by GoogleCodeExporter** [comment]
```
slightly related also to this one
```

-----
Original comment by `vittorio...@gmail.com` on 27 Mar 2011 at 10:22

**Comment by GoogleCodeExporter** [comment]
```
5. 'w' is reported as 'z' and viceversa outside chat event handling on Italian 
keyboards
```

-----
Original comment by `vittorio...@gmail.com` on 8 Jul 2011 at 7:24

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `sheepyluva` on 9 Jul 2011 at 11:51
* Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `vittorio...@gmail.com` on 30 Nov 2011 at 2:47

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `vittorio...@gmail.com` on 16 Apr 2013 at 2:23
* Changed title: **Tacker for keyboard layout problems**
* Added labels: **Type-Tracker**
* Removed labels: **Type-Defect**, **Priority-Medium**
* Now blocked on: #111, #120, #138, #142, #148, #152, #155, #317

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `sheepyluva` on 16 Apr 2013 at 7:11
* Changed title: **Tracker for keyboard layout problems**

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `vittorio...@gmail.com` on 13 May 2013 at 8:32

**Comment by GoogleCodeExporter** [comment]
```
Issue 496 has been merged into this issue.
```

-----
Original comment by `vittorio...@gmail.com` on 16 May 2013 at 8:24

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `vittorio...@gmail.com` on 16 May 2013 at 8:25
* Now blocked on: #479

**Comment by GoogleCodeExporter** [comment]
```
Issue 774 has been merged into this issue.
```

-----
Original comment by `kyberneticist@gmail.com` on 21 Jan 2014 at 3:09
